pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -304,6 +305,18 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		},
 		{
 			ObjectMeta: kapi.ObjectMeta{
+				Name: ImageAuditorRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					APIGroups: []string{imageapi.GroupName},
+					Verbs:     sets.NewString("get", "list", "watch", "patch", "update"),
```

Comment:
> I would not expect an auditor role to have any write permissions. Is there a different name we could use?

This is a thing that vets images to indicate whether they're good or bad.  Alternate name suggestions welcome.  I thought of this one and `image-inspector`, but thought that `auditor` was seemed like a better fit.